x/gov/abci_test.go

```diff
@@ -1,3 +1,5 @@
+// TODO update this file
```

Comment:
@MSalopek Is this TODO from the backport or from you?

Reply:
It's from the original PR, i didnt add it

Reply:
Removed